src/bindings/rust/README.md

```
@@ -62,4 +62,18 @@ Note that multithreading is disabled because NIXL might deadlock.
+script finds a CUDA toolkit, searched for under `CUDA_PATH`, `CUDA_HOME`, then
+`/usr/local/cuda`. Without one they are absent from the suite and nothing links
+against CUDA. With a toolkit but no device they skip at run time.
+
+`test_prep_mem_view_remote` additionally needs a device-capable RDMA lane, which
+UCX only offers over accelerated IB, so it is opt-in:
+
+```bash
+NIXL_TEST_DEVICE_LANE=1 cargo test --test tests mem_view
+```
+
+It is not skipped on failure, so where it runs, a failure is a real one.
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)